v2/tools/generator/internal/armconversion/shared.go

```diff
@@ -194,3 +189,20 @@ func removeEmptyStatements(stmts []dst.Stmt) []dst.Stmt {
 
 	return result
 }
+
+const (
+	ConversionTag        = "conversion"
+	NoARMConversionValue = "noarmconversion"
+)
+
+func skipNonConvertablePropertyHandler(
```

Comment:
I'd suggest having a method name which makes more sense in respect to skipping ARM conversion. Something like  `skipArmConversionPropertyHandler` or `noArmConversionPropertyHandler` ?

Reply:
I agree on improving the name, came up with `skipPropertiesFlaggedWithNoARMConversion`, inspired by your suggestions.